src/index.html

```diff
@@ -5,9 +5,49 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div class="card"
+    data-qa="card">
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="Apple"
+      >
+      <h1 class="card__productName">
+        APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h1>
```

Comment:
Will be better if you use another level tag here
```suggestion
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina
        <br>
        5K Monoblock (MNED2UA/A)
      </h2>
```